File: code/DAL/models/PlacementOfATeacher.cs
using System;
using System.Collections.Generic;

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
#nullable disable

namespace DAL.models
{
    public partial class PlacementOfATeacher
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Column("IdPlacementOfATeacher")]

        public int IdPlacementOfATeacher { get; set; }
        public int? DayInWeek1M { get; set; }
        public int? DayInWeek1A { get; set; }
        public int? DayInWeek2M { get; set; }
        public int? DayInWeek2A { get; set; }
        public int? DayInWeek4M { get; set; }
        public int? DayInWeek4A { get; set; }
        public int? DayInWeek3M { get; set; }
        public int? DayInWeek3A { get; set; }
        public int? DayInWeek5M { get; set; }
        public int? DayInWeek5A { get; set; }
        public int? DayInWeek6M { get; set; }

 
        public int TeacherId { get; set; }
        public virtual User UserTeacher { get; set; }
    }
}

File: myProject/myProject/src/app/classes/placementOfATeacher.ts
import { Classes } from "./Classes";
import { User } from "./Users";

export interface PlacementOfATeacher {
  IdPlacementOfATeacher: number;
  dayInWeek1M :number;
  dayInWeek1MStr :string;
  dayInWeek1A :number;
  dayInWeek1AStr :string;
  dayInWeek2M :number;
  dayInWeek2MStr :string;
  dayInWeek2A :number;
  dayInWeek2AStr :string;
  dayInWeek4M :number;
  dayInWeek4MStr :string;
  dayInWeek4A :number;
  dayInWeek4AStr :string;
  dayInWeek3M :number;
  dayInWeek3MStr :string;
  dayInWeek3A :number;
  dayInWeek3AStr :string;
  dayInWeek5M :number;
  dayInWeek5MStr :string;
  dayInWeek5A :number;
  dayInWeek5AStr :string;
  dayInWeek6M :number;
  dayInWeek6MStr :string;
  teacherId: number;
  userTeacher: User;
}
